src/index.html

```diff
@@ -8,6 +8,14 @@
   <link rel="stylesheet" href="./styles/main.css">
 </head>
 <body>
-  <h1>Antisnake</h1>
+  <!-- <h1>Antisnake</h1> -->
+  <div class="container">
+    <div class="container__item container__item--first"></div>
```

Comment:
where do you use class .container__item--first? you don't need these classes(container__item--first,container__item--second etc)

Reply:
just forgot to del it